src/ndb.c

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright (C) 2016 Swift Navigation Inc.
+ * Contact: Roman Gezikov <rgezikov@exafore.com>
+ *
+ * This source is subject to the license found in the file 'LICENSE' which must
+ * be be distributed together with this source. All other rights reserved.
+ *
+ * THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF ANY KIND,
+ * EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.
+ */
+
+#include <string.h>
+#include "ndb.h"
+
+#include "ndb_p1_section.h"
+#include "ndb_p3_section.h"
+
+ndb_file_version_t ndb_file_version = 0x00000001;
```

Comment:
`GIT_VERSION` is #defined to a string like `v0.21-77-g33f9988-dirty` for non-release builds and something like `v0.21` for release builds.

I doubt you need it, but there is also a read-only setting which holds the version:
`READ_ONLY_PARAMETER("system_info", "firmware_version", GIT_VERSION, TYPE_STRING);`